dev/tidb-in-kubernetes/faq.md

```diff
@@ -20,3 +20,60 @@ The default time zone setting for each component container of a TiDB cluster in
 
     * In the `values.yaml` file of the TiDB cluster, modify `timezone` settings in the `values.yaml` file of the TiDB cluster. For example, you can set it to `timezone: Asia/Shanghai` and then upgrade the TiDB cluster.
     * Refer to [Time Zone Support](/how-to/configure/time-zone.md) to modify TiDB service time zone settings.
+
+## Can HPA or VPA be configured on TiDB components?
+
+Currently, TiDB cluster does not support HPA (Horizontal Pod Autoscaling) or VPA (Vertical Pod Autoscaling), because it is difficult to achieve autoscaling on applications with state such as database. Autoscaling can not be achieved merely by the monitoring data of CPU and memory.
+
+## What scenarios require manual intervention when I use TiDB Operator to orchestrate TiDB clusters?
+
+Except for the operation of the Kubernetes cluster itself, TiDB Operator has the following two scenarios that might require manual intervention:
+
+* Adjusting the cluster after the auto-failover of TiKV. Detailed reference: [Auto-Failover](tidb-in-kubernetes/maintain/auto-failover.md)
+* Maintaining or dropping the specific Kubernetes nodes. Detailed reference: [Maintaining Nodes](tidb-in-kubernetes/maintain/kubernetes-node.md)
+
+## What is the recommended deployment topology when I use TiDB Operator to orchestrate a TiDB cluster on a public cloud?
+
+To achieve high availability and data security, it is recommended for you to deploy at least three Available Zones in a TiDB cluster in a production environment.
```

Comment:
```suggestion
To achieve high availability and data security, it is recommended that you deploy the TiDB cluster in at least three availability zones in a production environment.
```
See [Running in multiple zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/)